librz/include/rz_types.h

```diff
@@ -497,16 +497,26 @@ static inline void *rz_new_copy(int size, void *data) {
 #endif
 #else
 #ifdef _MSC_VER
-#ifdef _WIN64
+#if defined(_M_X64) || defined(_M_AMD64)
 #define RZ_SYS_ARCH   "x86"
 #define RZ_SYS_BITS   (RZ_SYS_BITS_32 | RZ_SYS_BITS_64)
 #define RZ_SYS_ENDIAN 0
```

Comment:
I think Meson could get endian value from `.endian()` function, see https://mesonbuild.com/Reference-tables.html
cc @ret2libc 
Maybe we should drop those and use it instead.

Reply:
It could be done in a separate PR though, obviously.